### PR TITLE
Klarna API Configuration Merchant ID 

### PR DESCRIPTION
**Pull request by ccondra**
## Purpose of this pull request

This pull request (PR) corrects an instruction in our Klarna API configuration.  In our UI, the label for the "Klarna API Username (UID") field is "Merchant ID".  However, in Klarna, the Merchant ID   
Is a different field.  This  PR corrects the instructions so merchants now enter their Klarna API Username (UID) in the Magento > Klarna API configuration Merchant ID field.

1. Corrected field desc in [Klarna](https://docs.magento.com/m2/ee/user_guide/configuration/sales/klarna.html) Config Reference.

Changed field desc of Merchant ID to the API Username (UID) from your Klarna credentials.
For consistency, also changed Password to the API password from your Klarna credentials. (rather than account, because an account supports multiple API users)

2. Corrected instructions in [Setting Up Klarna](https://docs.magento.com/m2/ee/user_guide/payment/klarna-setup.html)

Corrected our instructions so merchants must enter their Klarna API Username (UID) value in the Magento Klarna API configuration Merchant ID field. Also removed some unnecessary instructions, moved a couple of screenshots, and simplified the numbering.

3. Updated [Change Log](https://docs.magento.com/m2/ee/user_guide/magento/change-log.html).

If Magento Commerce, please specify:

- [x] Commerce only?
- [x] Commerce with B2B?

## Affected documentation pages

- [Klarna](https://docs.magento.com/m2/ee/user_guide/configuration/sales/klarna.html) (configuration reference)
- [Setting Up Klarna](https://docs.magento.com/m2/ee/user_guide/payment/klarna-setup.html)
- [Change Log](https://docs.magento.com/m2/ee/user_guide/magento/change-log.html)

## Additional information

Staging environment;  #447

### CODE REVIEW

File: src/payment/klarna-setup.md [hunk 1]
```diff
@@ -31,96 +31,91 @@ _Klarna Merchant Portal_
    - On the Home page of your account under **Preferences**, click  **Settings**.
```
Comment (hickskenh) [inline src/payment/klarna-setup.md:31]:
```suggestion
   - On the Home page of your account under **Preferences**, click **Settings**.
```

File: src/payment/klarna-setup.md [hunk 2]
```diff
@@ -31,96 +31,91 @@ _Klarna Merchant Portal_
...
    - In the upper-left corner of the page, click the **Menu** icon. Then in the left sidebar under **Preferences**, click **Settings**.
 
-   Then, do the following:
+1. On the **API Credentials** tab, click <span class="btn">Generate new API credentials</span>. Then when prompted, click **Create credentials**.
```
Comment (hickskenh) [inline src/payment/klarna-setup.md:34]:
```suggestion
1. On the **API Credentials** tab, click <span class="btn">Generate new API credentials</span>. When prompted, click **Create credentials**.
```

File: src/payment/klarna-setup.md [hunk 3]
```diff
@@ -31,96 +31,91 @@ _Klarna Merchant Portal_
...
+   ![Generate API Credentials]({% link images/images/payments-klarna-dashboard-playground-settings.png %}){: .zoom}
+   _Generate API Credentials_
 
 ## Step 2: Configure Magento
 
-1. Open another browser tab, and log in to the Admin of your Magento store.
+1. In a new browser tab, log in to the Admin of your Magento store.
 
-1. On the Admin sidebar, choose **Stores**. Then under **Settings**, choose **Configuration**.
+1. On the Admin sidebar, go to **Stores** > `Settings`> **Configuration**.
```
Comment (hickskenh) [inline src/payment/klarna-setup.md:49]:
```suggestion
1. On the Admin sidebar, go to **Stores** > `Settings` > **Configuration**.
```

File: src/payment/klarna-setup.md [hunk 4]
```diff
@@ -31,96 +31,91 @@ _Klarna Merchant Portal_
...
-   - To generate a record of interactions while testing the integration, set **Logging** to `Enable`.
+1. Enter the **Password** from your Klarna API credentials.
 
-     {:.bs-callout-info}
-     Set **Logging** to `Disabled` before going live with Klarna in your production store.
+1. To use the Klarna Playground for test transactions, set **Mode** to `Playground`.
 
-     ![API Configuration]({% link images/images/config-sales-payment-methods-klarna-api.png %}){: .zoom}
-     [_API Configuration_]({% link configuration/sales/klarna.md %})
+1. To generate a record of interactions while testing the integration, set **Logging** to `Enable`.
```
Comment (hickskenh) [inline src/payment/klarna-setup.md:80]: is it enabled and disabled or enable and disable?  the next line includes the "d"

File: src/payment/klarna-setup.md [hunk 5]
```diff
@@ -31,96 +31,91 @@ _Klarna Merchant Portal_
...
+1. If you want Klarna to prepare a set of personalized credit options for each customer, set **Enable Data sharing** to `Yes`. Then, set **Enable Data sharing on load** to one of the following:
 
-1. If you want to stop here, tap <span class="btn">Save Config</span>.
+   |**No** |Customer billing and shipping address information is shared with Klarna after the transaction is authorized. If a shipping address error is encountered during authorization, the customer is instructed to return to the previous step and correct the problem.|
+   |**Yes** |Customer billing and shipping address information is shared with Klarna when the payment method is selected during the checkout process.|
 
-1. When prompted, click the [Cache Management]({% link system/cache-management.md %}) link at the top of the page.  Then, refresh all invalid caches.
+1. Enter a **Sort Order** number to determine the position of Klarna in the list of payment methods during checkout. (0 = first, 1 = second, 2 = third, and so on.)
 
 ## Step 3: Complete the Design Settings **(Optional)**
```
Comment (hickskenh) [inline src/payment/klarna-setup.md:106]: i don't think i've seen this.  bold on a header?  isn't the header already bold?  maybe ital?

File: src/payment/klarna-setup.md [hunk 6]
```diff
@@ -31,96 +31,91 @@ _Klarna Merchant Portal_
...
-- Border radius
+   ![Klarna Payments Design]({% link images/images/config-sales-payment-methods-klarna-payments-design.png %}){: .zoom}
+   [_Design_]({% link configuration/sales/klarna.md %})
 
-![Klarna Payments Design]({% link images/images/config-sales-payment-methods-klarna-payments-design.png %}){: .zoom}
- [_Design_]({% link configuration/sales/klarna.md %})
+1. When complete, click <span class="btn">Save Config</span>.
 
-When complete, click <span class="btn">Save Config</span>. Then when prompted, click the [**Cache Management**]({% link system/cache-management.md %}) link at the top of the page to refresh all invalid caches.
+1. Click the [**Cache Management**]({% link system/cache-management.md %}) link in the message at the top of the page, and refresh all invalid caches.
```
Comment (hickskenh) [inline src/payment/klarna-setup.md:121]: should this be "Click Cache Management in the..."?

File: src/payment/klarna-setup.md [hunk 7]
```diff
@@ -15,7 +15,7 @@ _Klarna Merchant Portal_
 1. Do the following from the Admin of your Magento installation:
 
    - On the Admin sidebar, go to **Stores** > Settings > **Configuration**.
```
Comment (hickskenh) [inline src/payment/klarna-setup.md:17]:
```suggestion
   - On the Admin sidebar, go to **Stores** > _Settings_ > **Configuration**.
```

File: src/payment/klarna-setup.md (hunk 4 above)
Comment (ccondra) [reply src/payment/klarna-setup.md:80]: Enable is correct here.

File: src/payment/klarna-setup.md (hunk 5 above)
Comment (ccondra) [reply src/payment/klarna-setup.md:106]: I don't think it needs either -- the parens are enough. Thanks!

File: src/payment/klarna-setup.md (hunk 6 above)
Comment (ccondra) [reply src/payment/klarna-setup.md:121]: Yes, that's better.

**Comment by ccondra** [comment]
running tests

**Pull request merged**